plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1153,7 +1153,7 @@ public void addColumn(ConnectorSession session, ConnectorTableHandle tableHandle
                     columnsBuilder.build(),
                     partitionColumns,
                     columnComments.buildOrThrow(),
-                    columnNullability.buildOrThrow(),
+                    columnsNullability.buildOrThrow(),
```

Comment:
shouldn't this be a separate commit ?

Reply:
I think it's related: we allow adding NOT NULL for empty tables
(previously it was ignored)

i think it;s fine to keep as one commit